src/components/locationForm/components/RadioGroupInput.jsx

```diff
@@ -4,12 +4,21 @@ import styled from '@datapunt/asc-core';
 
 const RadioLabelStyle = styled(Label)`
   font-weight: normal;
+
+  & > :first-child {
+    margin: 0;
+    padding-top: 5px;
```

Comment:
`themeSpacing()`?

Reply:
not appropriate here, is not aligning with other elements but with the native radio control. Will be replaced later by the styled radio button from `ASC`